packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec

```diff
@@ -85,8 +85,10 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/spec
 
 %changelog
-* Tue Apr 13 2021 ianballou <ianballou67@gmail.com> 3.10.0-1
+* Fri May 21 2021 Evgeni Golov 3.10.0-0.1
```

Comment:
Otherwise RPMlint will yell at me for a changelog "going backwards" and I am in no mood to rebuild this here and then also in 4.1/nightly for no good reason.